tests/subscription_server/mod.rs

```diff
@@ -116,6 +116,6 @@ pub struct SubscriptionRoot {
 impl SubscriptionRoot {
     async fn books(&self, _mutation_type: MutationType) -> impl Stream<Item = BookChanged> {
         println!("Subscription received");
-        BroadcastStream::new(self.channel.subscribe()).filter_map(|r| r.ok())
+        BroadcastStream::new(self.channel.subscribe()).filter_map(std::result::Result::ok)
```

Comment:
I don't think this is an improvement tbh.

Reply:
Clippy suggestion. Keep it or?

Reply:
Whatever tbh